data/json/mapgen_palettes/deserter_city.json

```diff
@@ -0,0 +1,192 @@
+      "≥": {
+        "chunks": [
+          [ "8x8_meeting_room_r_1", 1 ],
+          [ "8x8_meeting_room_r_2", 1 ],
+          [ "8x8_meeting_room_r_3", 1 ],
+          [ "8x8_meeting_room_r_4", 1 ]
+        ]
+      },
+      "(": {
+        "chunks": [
+          [ "14x14_office_slide_sw", 1 ],
+          [ "14x14_office_glass_sw", 1 ],
+          [ "14x14_office_cubicle_sw", 1 ],
+          [ "14x14_office_crowded_sw", 1 ],
+          [ "14x14_office_generic_sw", 1 ]
+        ]
+      },
+      ")": {
+        "chunks": [
+          [ "14x14_office_slide_se", 1 ],
+          [ "14x14_office_glass_se", 1 ],
+          [ "14x14_office_generic_se", 1 ],
+          [ "14x14_office_cubicle_se", 1 ],
+          [ "14x14_office_crowded_se", 1 ]
+        ]
+      },
+      "{": {
+        "chunks": [
+          [ "14x14_office_glass_nw", 1 ],
+          [ "14x14_office_cubicle_nw", 1 ],
+          [ "14x14_office_generic_nw", 1 ],
+          [ "14x14_office_slide_nw", 1 ],
+          [ "14x14_office_crowded_nw", 1 ]
+        ]
+      },
+      "}": {
+        "chunks": [
+          [ "14x14_office_glass_ne", 1 ],
+          [ "14x14_office_cubicle_ne", 1 ],
+          [ "14x14_office_generic_ne", 1 ],
+          [ "14x14_office_slide_ne", 1 ],
+          [ "14x14_office_crowded_ne", 1 ]
+        ]
+      },
+      "1": {
+        "chunks": [
+          [ "5x5_cabinet_toilet_l", 2 ],
+          [ "5x5_cabinet_storage0_l", 1 ],
+          [ "5x5_cabinet_storage1_l", 1 ],
+          [ "5x5_cabinet_storage2_l", 1 ],
+          [ "5x5_cabinet_cleaning0_l", 2 ],
+          [ "5x5_cabinet_cleaning1_l", 2 ]
+        ]
+      },
+      "2": {
+        "chunks": [
+          [ "5x5_cabinet_toilet_r", 2 ],
+          [ "5x5_cabinet_storage0_r", 1 ],
+          [ "5x5_cabinet_storage1_r", 1 ],
+          [ "5x5_cabinet_storage2_r", 1 ],
+          [ "5x5_cabinet_cleaning0_r", 2 ],
+          [ "5x5_cabinet_cleaning1_r", 2 ]
+        ]
+      },
+      "!": {
+        "chunks": [
+          [ "5x5_corridor_office_ne", 2 ],
+          [ "5x5_corridor_benches_ne", 1 ],
+          [ "5x5_corridor_sofa_ne", 1 ],
+          [ "5x5_corridor_office0_se", 1 ],
+          [ "5x5_corridor_office1_se", 1 ],
+          [ "5x5_corridor_sofas_se", 2 ]
+        ]
+      },
+      "¡": {
+        "chunks": [
+          [ "5x5_cabinet_office0_n", 1 ],
+          [ "5x5_cabinet_office1_n", 1 ],
+          [ "5x5_cabinet_sec_n", 1 ],
+          [ "5x5_cabinet_sofa_n", 1 ]
+        ]
+      },
+      "3": {
+        "chunks": [
+          [ "6x6_cabinet_office_n", 1 ],
+          [ "6x6_cabinet_glass_ne", 1 ],
+          [ "6x6_cabinet_computer_ne", 1 ],
+          [ "6x6_cabinet_files_ne", 1 ]
+        ]
+      },
+      "4": {
+        "chunks": [
+          [ "5x5_corridor_register_ne", 1 ],
+          [ "5x5_corridor_office_ne", 1 ],
+          [ "5x5_corridor_benches_ne", 1 ],
+          [ "5x5_corridor_sofa_ne", 1 ]
+        ]
+      }
+    }
+  }
```

Comment:
This doesn't seem to be used by anything. I'm pretty sure there's also a lot of terrain and furniture here that should not be included in the palette.